runtime/compiler/control/CompilationThread.cpp

```diff
@@ -498,7 +498,7 @@ bool TR::CompilationInfo::shouldDowngradeCompReq(TR_MethodToBeCompiled *entry)
                  getMethodQueueSize() >= TR::Options::_qszThresholdToDowngradeOptLevelDuringStartup) ||
                  // Downgrade if AOT and startup, 
                 (TR::Options::getCmdLineOptions()->sharedClassCache() &&
-                 _jitConfig->javaVM->phase == J9VM_PHASE_STARTUP &&
```

Comment:
I understand that we don't want to downgrade during startup as per the intent of this change. 
I'm curious why we changed the condition such that it would downgrade when we are not in startup though. 
i.e. why do we want to change anything if we are not in startup phase ?